svelte/components/Nft/NftLine.svelte

```diff
@@ -0,0 +1,251 @@
+<script lang="ts">
+  import type { Readable } from "svelte/store";
+
+  import type { NftType } from "lib/ktypes";
+  import {
+    getShortAddress,
+    nftUrl,
+    explorerCollectionUrl,
+    nftDescription,
+    nftDescriptionShort,
+    nftName,
+    nftOpenSeaUrl,
+    addressSame,
+    textShort,
+    explorerAddressLink,
+    kredeumNftUrl,
+    getNetwork
+  } from "lib/kconfig";
+  // import { nftKey } from "lib/kconfig";
+  import { nftGetImageLink } from "lib/knft-get-metadata";
+  import { onMount } from "svelte";
+
+  import { nftStore } from "stores/nft/nft";
+  import { shortcode } from "helpers/shortcodes";
+  import NftTransfer from "./NftTransfer.svelte";
+
+  /////////////////////////////////////////////////
+  //  <Nft {chainId} {address} {tokenID} {account}? {index}? {more}? {platform}? />
+  // Display NFT
+  /////////////////////////////////////////////////
+  export let chainId: number;
```

Comment:
NftLine should be similar to NftGrid,  i.e. being a almost pure viewer components (no or almost no typescript in it), taking Nft as prop

Reply:
done